### PR TITLE
Add explicity `--unit` to automated tests that run on `--local-changes`

### PR DESCRIPTION
**Pull request by danielrozenberg**
Fixed that issue noticed by Raghu in https://github.com/ampproject/amphtml/pull/21407#pullrequestreview-223524623

### CODE REVIEW

File: build-system/default-pre-push [hunk 1]
```diff
@@ -25,8 +25,8 @@
 GREEN() { echo -e "\033[0;32m$1\033[0m"; }
 CYAN() { echo -e "\033[0;36m$1\033[0m"; }
 
-GULP_LINT_LOCAL="gulp lint --local-changes"
-GULP_TEST_LOCAL="gulp test --local-changes --headless"
+GULP_LINT_LOCAL="gulp lint --unit --local-changes"
```
Comment (rsimha) [inline build-system/default-pre-push]: This is `gulp lint`. No need for `--unit`.

Comment (danielrozenberg) [reply build-system/default-pre-push]: lol, trigger-happy search & replace. Done.

**Comment by cathyxz** [comment]
What happens if you don't add `--unit` to `--local-changes`? Is the heuristic smart enough to run the correct set of runtime tests?

**Comment by danielrozenberg** [comment]
@cathyxz yes, it'll still run the unit tests without it. This is for the test status bot (followup for #21407) to report the correct type of testing being done when running `--local-changes`

**Comment by cathyxz** [comment]
Super cool. Thanks!

**Pull request merged**